locale/es_MX/LC_MESSAGES/clr-installer.po

```diff
@@ -321,7 +321,7 @@ msgid "Welcome to Clear Linux* OS Desktop Installation"
 msgstr "Bienvenido a la instalación del sistema operativo Clear Linux* para computadoras de escritorio"
 
 msgid "Thank you for choosing Clear Linux* OS"
-msgstr "Gracias por elegir el sistema operativo Clear Linux*"
+msgstr "Gracias por \nelegir el \nsistema \noperativo \nClear Linux*"
```

Comment:
Why are they adding embedded newlines to the text? I thought we had worked hard to remove these?

Reply:
I noticed this. But It appeared fine on the page. So ignored. Maybe we can ask them.